core/src/interface.ts

```diff
@@ -476,6 +476,7 @@ export interface MediaLibraryExternalLibrary {
 export interface MediaLibraryInternalOptions {
   allow_multiple?: boolean;
   choose_url?: boolean;
+  config?: MediaLibraryOptions;
```

Comment:
Since we control the internal library config options, I would prefer we use a more specific type here. `MediaLibraryOptions` allows for pretty much anything to be added to the config.

```suggestion
  config?: {
    max_file_size?: number;
  };
```